src/scss/utilities/reset.scss

```diff
@@ -58,3 +60,12 @@ button {
 dd, dt {
 	display: inline-block;
 }
+
+a {
+  text-decoration: none;
```

Comment:
I wouldn't add anything to this file, basically because it is a ```vendor``` file. I'd create a new one adding all our reset things.